sentences/ru/shopping_list_HassShoppingListAddItem.yaml

```diff
@@ -0,0 +1,11 @@
+language: ru
+intents:
+  HassShoppingListAddItem:
+    data:
+      - sentences:
+          - "добав(ить|ь) <item> в [мой] список покупок"
+          - "нужно купить <item> "
```

Comment:
Not sure how big of a deal that trailing space is. It doesn't seem to cause the `<item>` to have a leading space anyway, which is why I had to extract that particular sentence into a separate test.